wpilib/src/pwm.rs

```diff
@@ -0,0 +1,233 @@
+// Copyright 2018 First Rust Competition Developers.
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use sensor_util;
+use wpilib_sys::*;
+
+pub struct PWM {
```

Comment:
I think its a good convention to reorder definitions so intrinsic `impl`s appear directly after the struct definition.